install.sh

```diff
@@ -33,18 +33,18 @@ fi
 echo "Gateway configuration:"
 
 # Try to get gateway ID from MAC address
-# First try eth0, if that does not exist, try wlan0 (for RPi Zero)
-GATEWAY_EUI_NIC="eth0"
-if [[ `grep "$GATEWAY_EUI_NIC" /proc/net/dev` == "" ]]; then
-    GATEWAY_EUI_NIC="wlan0"
-fi
 
-if [[ `grep "$GATEWAY_EUI_NIC" /proc/net/dev` == "" ]]; then
+# Get first non-loopback network device that is currently connected
+NICS=$(ip link 2>&1 | grep "state UP" | grep -v LOOPBACK | sed -E 's/^[0-9]+: ([0-9a-z]+): .*/\1/')
```

Comment:
`"state UP"` sounds like it's locale-dependent, I'm not totally sure thou, I think `UP` is always present regardless of the language, but `state` maybe not.

Reply:
I tested it with German and Esperanto locales (which are both installed on my system) and the output always said `state`.

Reply:
🤩